### ISSUE TITLE
TypeError: Could not handle Node of kind: 'JSDocSeeTag'

### ISSUE THREAD

**Issue by antikvarBE** [wessberg/rollup-plugin-ts#125]
- Version: 1.3.8
- Rollup Version: 2.33.3
- Operating System and version (if applicable): MacOS BigSur 11.0.1
- Node Version (if applicable): 15.2.1
- Does it work with `tsc` (if applicable):

### Expected Behavior
I used to be able to use JSDoc native tag @see and compile my bundle normally.

### Actual Behavior
Now I can't anymore :)

```
[!] (plugin Typescript) TypeError: Could not handle Node of kind: 'JSDocSeeTag'
TypeError: Could not handle Node of kind: 'JSDocSeeTag'
    at executeCloneNode (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:1281:8)
    at nextNode (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:302:16)
    at /Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:350:39
    at Array.map (<anonymous>)
    at nextNodes (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:350:27)
    at Object.nextNodes (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:291:84)
    at v4Strategy (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-js-doc-comment.ts:34:48)
    at cloneJsDoc (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-js-doc-comment.ts:8:10)
    at executeCloneNode (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:1118:10)
    at nextNode (/Users/antikvar/Projects/Zvook/Development/ui-kit/node_modules/@wessberg/ts-clone-node/src/clone-node/clone-node.ts:302:16)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by ckknight** [comment]
I think this is related: https://github.com/wessberg/ts-clone-node/pull/4

**Comment by wessberg** [comment]
Hey there,
Sorry for the late reply.

This has been fixed as part of [v1.3.10](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.3.10)
